src/python/pants/engine/process.py

```diff
@@ -266,6 +266,7 @@ class InteractiveProcess:
     input_digest: Digest
     run_in_workspace: bool
     hermetic_env: bool
+    capture_signals: bool
```

Comment:
I think this might be ambiguous: _who_ is capturing signals? Pants, or the process? Another way of framing things: the process would presumably already have all signals; Pants is the magical thing that's capturing things. (But I could also see someone thinking the opposite.)

I'm trying to think of a more explicit name. Maybe something like this:

* `pants_captures_signals=True`
* `handle_signals=False`
* `process_handles_signals=False`

At a minimum, it'd be good to have docstring about this.

Reply:
I renamed this to `forward_signals_to_process` and added a docstring.

Reply:
I love it. Thanks!